keycloak-next/README.md

```diff
@@ -0,0 +1,64 @@
+# Keycloak Helm Deployment
+## Overview
+
+This repository contains configuration files for deploying Keycloak on Kubernetes using the official Bitnami Legacy Keycloak Helm chart.
+
+Keycloak provides identity and access management (IAM) with support for Single Sign-On (SSO), OAuth2, OpenID Connect, and SAML.
+
+
+# Prerequisites
+
+Ensure the following are installed:
+
+Kubernetes cluster (K8s v1.23+ recommended)
+
+Helm v3+
+
+kubectl configured to access your cluster
+
+Bitnami Sealed Secrets
+
+External database (PostgreSQL)
+
+
+## Install Keycloak
+```bash
+helm install keycloak ./keycloak-next --namespace keycloak -f values.yaml
+```
+
+## Upgrade Release
+```bash
+helm upgrade keycloak ./keycloak-next --namespace keycloak -f values.yaml
+```
+
+## Uninstall Keycloak
+```bash
+helm uninstall keycloak --namespace keycloak
```

Comment:
So @sahithya23 when we run this, it will use the current default kubectl context to deploy to the switched context env, right?